util/gconv/gconv.go

```diff
@@ -35,13 +34,6 @@ var (
 		"off":   {},
 		"false": {},
 	}
-
-	// StructTagPriority defines the default priority tags for Map*/Struct* functions.
-	// Note that, the `gconv/param` tags are used by old version of package.
-	// It is strongly recommended using short tag `c/p` instead in the future.
-	StructTagPriority = []string{
```

Comment:
As the `StructTagPriority` variable is mainly for converting feature, it is recommended to keep it in package `gconv`.
And also, do not remove public variable as it might lead incompatible with old versions.